src/v2/view-builder/views/SuccessView.js

```diff
@@ -1,14 +1,24 @@
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import $ from 'jquery';
 
 const Body = BaseForm.extend({
   title () {
     return  'You will be redirected';
   },
   noButtonBar: true,
-  initialize (options) {
+  initialize () {
     BaseForm.prototype.initialize.apply(this, arguments);
-    options.appState.trigger('saveForm', options.model);
+    // TODO OKTA-250473
+    // Form post for success redirect
+    const data = {
+      stateHandle: this.options.appState.get('currentState').stateHandle
+    };
+    return $.post({
+      url: '/idp/idx/login/token/redirect',
```

Comment:
hmm, sorry I forgot we didn't expose the remediation form details.. hard-code the URL is bad..
how about hack in `actionsTransformer.js`?

Reply:
yeah, I just added this since we don't have access to the post url here. I'll try to move this elsewhere

Reply:
How about we don't omit the `href` attribute in the responseTransformer, so that it can be accessed via currentState?